doc/source/learning/install-caasp.rst

```diff
@@ -0,0 +1,19 @@
+CaaSP v3 Install
+================
+
+At this moment we need CaaSP v3 for our deployments. This is our SUSE supported Kubernetes deployment. The minimum number of nodes needed for the install is 4: 1 admin node, 1 kubernetes master node and 2 kubernetes worker nodes.
+
+To install CaaSP you can find a `deployment guide`_ and a `quick install`_ guide on SUSE online documentation.
+
+.. _deployment guide: https://www.suse.com/documentation/suse-caasp-3/singlehtml/book_caasp_deployment/book_caasp_deployment.html
+.. _quick install: https://www.suse.com/documentation/suse-caasp-3/singlehtml/book_caasp_installquick/book_caasp_installquick.html
+
+For development environments we are deploying CaaSP on top of OpenStack VMs. The best way to deploy several VMs together is through Heat teamplates. You can find a `specific section`_ on how to do that in our online documentation.
```

Comment:
Should be mentioned that this goal is achieved by executing `run.sh deploy_caasp` step.

Reply:
Agreed. In here, or in the overview.

Reply:
I move this part to the overview, adding these information